tools/cibadmin.c

```diff
@@ -458,6 +536,46 @@ main(int argc, char **argv)
         exit_code = crm_errno2exit(rc);
     }
 
+#if ENABLE_ACL
+    if (output != NULL && acl_eval) {
+        xmlDoc *acl_evaled_doc;
+        rc = pcmk_acl_evaled_as_namespaces(PCMK_ACL_CRED_USER, acl_cred,
+                                           output->doc, &acl_evaled_doc);
+        if (rc > 0) {
+            free_xml(output);
+            if (acl_eval_how != acl_eval_ns_full) {
+                xmlChar *rendered = NULL;
+                int rendered_len = 0;
+                enum pcmk__acl_render_how how = (acl_eval_how == acl_eval_ns_simple)
+                                                ? pcmk__acl_render_ns_simple
+                                                : (acl_eval_how == acl_eval_text)
+                                                ? pcmk__acl_render_text
+                                                : (acl_eval_how == acl_eval_color)
+                                                ? pcmk__acl_render_color
+                                                : /*acl_eval_auto*/ isatty(STDOUT_FILENO)
+                                                ? pcmk__acl_render_color
+                                                : pcmk__acl_render_text;
+                if (!pcmk__acl_evaled_render(acl_evaled_doc, how,
+                                             &rendered, &rendered_len)) {
+                    printf("%s\n", (char *) rendered);
+                } else {
+                    crm_crit("Could not render evaluated ACLs");
+                    crm_exit(CRM_EX_CONFIG);
+                }
+                output = NULL;
+            } else {
+                output = xmlDocGetRootElement(acl_evaled_doc);
+            }
+        } else if (rc == 0) {
+
+        } else if (rc < 0) {
+            crm_crit("Could not evaluated ACLs per request (%s, rc=%d)",
```

Comment:
spelling: evaluate

Tool errors should go to stderr rather than the log

Reply:
That's the matter of already discussed terrible discrepancy in using
progress-info means ambiguously.  In an ideal world, `crm_log_init`
would use the "is daemon" flag and continue to forward towards syslog,
file logs, journal or whatever else is configured for the daemons to
use.  For end interaction programs, it would either forward it to
`stderr` or to (perhaps later on) to a custom logging handler specific
to the program in question.

It would reolve all these nasty discrepancies once forever.
End result: relaxed programmers since they use a single familiar
interface carring just about single metadata item to be attached
(severity) and leaving everything else to the "middleware" to get
the routing right.

It's sad we are not there and there were even objections to that
natural model to be adopted.

Will use nasty explicit `fprintf` instead.